README.md

```diff
@@ -37,8 +77,114 @@ docker-compose up --build
+
+#### Dashboard
+
+The dashboard view is for developers to add, remove and manage instances.
+After creating an instance the developer recieves a simple link that can be shared, so that people easily can view the instance and give feedback.
+[Screenshot](#dashboard-image)
+
+#### Feedbacker
+
+The feedbacker view is where the users can use, view and comment on the UI. It consists of two main parts, the survey panel and the comment panel.
+[Screenshot](#feedbacker-image)
+
+#### Features
+
+A brief list of supported features
+* Automatically deploy your app from git, boh public repos and private Github repos supported
+* Alternatively use an already live site, simply submit the url
+* Easily discuss and cultivate ideas with theaded discussion, no blocking signup required
+* Tag specific elements to link your comment to a specific part of the UI
+* Comment anonymously on instances where you don't want to share your identity
+* Specify the feedback recieved by creating surveys with normal or multiple choice questions
+* Protect sensitive deployments with a passphrase
+* Boost productivity by integrating Feedbacker-Forum to slack with a SlackBot
+
+
+## Contributing
+
+The project is still in a quite early development stage and is actively being developed.
+Changes or additions can be proposed by creating issues or pull requests on github. Running a local development version can be setup as seen in [Setting Up](#Setting-up).
+
+### Running tests
+
+The project contains has style checking using [eslint][eslint] with a slightly modified [airbnb-config][airbnb-config].
+
+```bash
+# Run style checking for the frontend
+cd client/
+npm run lint
+
+# Then run style checking for the backend
+cd ../server/
+npm run lint
+```
+
+Tests are run separately for frontend and backend. Both sets of tests can be automatically run with npm
+
+```bash
+# Run tests for frontend
+cd client/
+npm run test
+
+# Then run tests for backend
+cd ../server/
+npm run test:api
+```
+
+The frontend test suite is done with [jest][jest] and [enzyme][enzyme]. The backend tests are currently only for the api-endpoints and are run with [mocha][mocha].
+
+## Credits
+
+This project was started during the course CS-C2130 Software Project at [Aalto University][aalto].
+The project was supported by [Sievo][sievo].
+The project was created by
+- [@bqqbarbhg](https://github.com/bqqbarbhg)
+- [@tobbe-j](https://github.com/tobbe-j)
+- [@joonalillfors](https://github.com/joonalillfors)
+- [@quvide](https://github.com/quvide)
+- [@sofiatengvall](https://github.com/sofiatengvall)
+- [@Kallemakela](https://github.com/Kallemakela)
+- [@shahruzmannan](https://github.com/shahruzmannan)
+- [@HenrikHelenius](https://github.com/HenrikHelenius)
+
+## License
+
+The project is licensed under the [MIT][MIT] license, see [LICENSE](LICENSE) for more details.
+
+## Screenshots
+
+#### Demo
+Demo video [Feedbacker-Forum](https://www.youtube.com/watch?v=bkxmN-j-ae0)
```

Comment:
Fake video embed like this: https://stackoverflow.com/questions/4279611/how-to-embed-a-video-into-github-readme-md